test/acceptance/public-api-test.js

```diff
@@ -286,31 +286,54 @@ describe('public api', function () {
     });
 
     it('returns whether the source has been fixed + an array of remaining issues with the provided template', async function () {
+      project.write({
+        app: {
+          templates: {
+            'application.hbs': '<div>FORBIDDEN</div>',
+          },
+        },
+      });
+      linter = new Linter({
+        console: mockConsole,
+        config: {
+          plugins: [require('../helpers/failure-plugin')],
+          rules: {
+            'fail-on-word': 'FORBIDDEN',
+          },
+        },
+      });
+
       let templatePath = project.path('app/templates/application.hbs');
       let templateContents = fs.readFileSync(templatePath, { encoding: 'utf8' });
-      let expected = [
-        {
-          column: 7,
-          line: 1,
-          endColumn: 18,
-          endLine: 1,
-          message: 'you must use double quotes in templates',
-          filePath: templatePath,
-          rule: 'quotes',
-          severity: 2,
-          source: "class='mb4'",
-        },
-      ];
 
       let result = await linter.verifyAndFix({
         source: templateContents,
         filePath: templatePath,
         moduleId: templatePath.slice(0, -4),
       });
 
-      expect(result.messages).toEqual(expected);
-      expect(result.output).toEqual(templateContents);
-      expect(result.isFixed).toEqual(false);
+      expect(result).toMatchInlineSnapshot(
+        { messages: [{ filePath: expect.any(String) }] },
```

Comment:
This line looks wrong. I believe `toMatchInlineSnapshot` should only have one argument.

Reply:
Nevermind, see https://github.com/ember-template-lint/ember-template-lint/pull/2143/files#r758654423